pkgs/generate.sh

```diff
@@ -0,0 +1,30 @@
+
+#!/bin/sh
```

Comment:
ah, didnt notice that, and bash likes to cheat and just treat "invalid" files as shell scripts for you